pages/guide.md

```diff
@@ -140,6 +141,31 @@ Where privacy is a requirement, OCSP transactions exchanged using HTTP MAY be pr
 
 Agencies are encouraged to operate OCSP and CRL services via hostnames specifically reserved for those services, so that other related information and functionality can be served securely and privately.
 
+### How can I avoid breaking XML clients that resolve external DTDs and XML entities over plain HTTP?
+
+In some specific cases, agencies may host DTDs or other XML entities that cause common XML clients to resolve system identifiers for DOCTYPEs and external entity declarations by fetching `http://` URIs.
+
+This might look like:
+
+```xml
+<!DOCTYPE PUBLIC "//name" SYSTEM "http://example.gov/dtds/example.dtd">
+```
+
+Or:
+
+```xml
+<!ENTITY entityname SYSTEM "http://example.gov/entities/logo.xml">
+```
+
+At this time, most common XML clients are unable to make TLS connections over HTTPS, or are unable to follow redirects from `http://` to `https://`.
```

Comment:
Many, not most, I'd say. libxml2 is an example of the former, java of the latter.